app/views/users/show.html.erb

```diff
@@ -1 +1,60 @@
-<%= @user.user_name %>
\ No newline at end of file
+<p id="notice"><%= notice %></p>
+<h1 class="title">My Events</h1>
+<hr>
+
+<h1 class="title">Created Events</h1>
+<table class="table">
+  <thead>
+    <tr>
+      <th>Title</th>
+      <th>Body</th>
+      <th>Date</th>
+      <th>Location</th>
+      <th>Creator</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% current_user.events.each do |event| %>
```

Comment:
This data fetching `current_user.events` should be called in your `users_controller#show`, just like the way you did it in your `events_controller#index action.